sections/03_categorize.md

```diff
@@ -47,53 +47,67 @@ regards to data availability and labeling.
 One of the general areas where deep learning methods have had substantial
 success has been in image analysis. Applications in areas of medicine that use
 imaging extensively are also emerging. Mammography has been one area with
-numerous contributions [@doi:10.1007/978-3-319-46723-8_13
-@doi:10.1007/978-3-319-24553-9_74 @doi:10.1101/095794 @doi:10.1101/095786]. In
-all of this work, the researchers must work around a specific challenge - the
-limited number of well annotated training images. To expand the number and
-diversity of images, the researchers have employed approaches where they employ
-adversarial examples [@doi:10.1101/095786] or first train towards human-created
-features before subsequent fine tuning [@doi:10.1007/978-3-319-46723-8_13]. The
-presence of a large bank of well-annotated mammography images would aid in the
-application of deep neural networks to this area. Though this strategy has not
-yet been employed in this domain, large collections of unlabeled images might
-first be used in an unsupervised context to construct high-quality feature
-detectors. Then the small number of labeled examples could be used for
+numerous contributions [@tag:Dhungel2015_struct_pred_mamm
+@tag:Dhungel2016_mamm @tag:Zhu2016_mult_inst_mamm @tag:Zhu2016_advers_mamm
+@tag:Dhungel2017_mamm_min_interv]. In all of this work, the researchers must
+work around a specific challenge - the limited number of well annotated
+training images. To expand the number and diversity of images, the researchers
+have employed approaches where they employ adversarial examples
+[@tag:Zhu2016_advers_mamm] or first train towards human-created features before
+subsequent fine tuning [@tag:Dhungel2016_mamm]. Adaptation to the medical image
+domain can further improved by combining in the latter approach with other
+machine learning techniques, for example, as a cascade of deep learning and
+random forest models [@tag:Dhungel2017_mamm_min_interv]. Using larger dataset
+of around 45,000 images, Kooi et al. [@tag:Kooi2016_mamm_lesions] demonstrated
+that deep neural networks can outperform the traditional CAD system at low
```

Comment:
It only gets used once from what I see - I'd say expand out the acronym at least, and define if it's valuable for readers to know the acronym as well.